R/SingleR.visualizations.R

```diff
@@ -0,0 +1,121 @@
+#' Plot a scatter plot of a single cell vs. a single reference cell/sample
+#'
+#' @param sc_data Similar to SinlgeR test input, a numeric matrix of single-cell expression values (usually
+#' log-transformed or otherwise variance-stabilized), where rows are genes and columns are cells.
+#' Alternatively, a \linkS4class{SingleCellExperiment} object containing such a matrix.
+#' @param cell_id a number indicating which single cell to use.
+#' Alternatively, the name of the cell.
+#' @param refdata Similar to SinlgeR training input, a numeric matrix of expression values. 
+#' Alternatively, a \linkS4class{SingleCellExperiment} object containing such a matrix.
+#' This should have the same rows as or a subset of the rows in \code{test}.
+#' @param sample_id a number of the sample to use
```

Comment:
My favored documentation style would be something like this:

```
@param sample_id Integer scalar specifying the index of the sample to use.
```

Note the caps and full stop, and the fact that I state the expected data type(s). (This is from the old days of working with Fortran code, where it simply wouldn't compile if you didn't give it the right type.)

Reply:
Implemented.